loadtest/control/actions.go

```diff
@@ -217,6 +217,24 @@ func CreatePost(u user.User) UserActionResponse {
 	return UserActionResponse{Info: fmt.Sprintf("post created, id %v", postId)}
 }
 
+// PatchPost updates a post.
+func PatchPost(u user.User) UserActionResponse {
+	post, err := u.Store().RandomPost()
```

Comment:
In a more simulative controller we might want to call something like `RandomPostInChannel(currentChannelId)`.
Just a thought for a potential follow-up PR.

Reply:
Sure, we can 👍

Reply:
After a second thought, I think @streamer45's proposal makes more sense. And, `RandomPost()` does not searches from the same team, if you want to edit a post, you might be do it in your current channel. Going to update accordingly.

Reply:
> RandomPost at least searches in the same team right ?

It doesn't but that's the kind of approach we always kept with `SimpleController`. Just trying to stress the server (make the API calls) without too much logic behind it. I don't think it's wrong, just not very realistic but we have `SimulController` for that.